.github/workflows/desktop-build.yml

```diff
@@ -208,28 +208,28 @@ jobs:
 
           - target: 10.14_Mojave
             os: macos-10.15 # runs on Catalina
-            xcode: 10.3 # allows compatibility with macOS 10.14
+            xcode: 11.3 # allows compatibility with macOS 10.14
```

Comment:
This will no longer target macOS 10.14, but 10.15

Unfortunately, Xcode 10.3 is the most recent (and only) option we have to keep support for 10.14